src/components/MaterialUI/DataDisplay/PredefinedElements/ActionModal.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Modal from "./../Modal";
+import ModalProps from "./../modalProps";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import sharedMessages from "../../../../sharedMessages";
+import { FormattedMessage } from "react-intl";
+
+const useStyles = makeStyles(theme => ({
+	actionPanel: {
+		float: "right",
+	},
+	message: {
+		maxWidth: "480px",
+		wordWrap: "normal",
+	},
+}));
+
+const ActionModal = ({
+	title,
+	message,
+	open,
+	actions, // Array of objects containing three properties: label, handler, isPrimary
+	backdropClickCallback,
+}) => {
+	const classes = useStyles();
+
+	const modalProps = new ModalProps();
+
+	const titleComponent = title || <FormattedMessage {...sharedMessages.confirmation} />;
+	const messageComponent = (
+		<div className={classes.message}>
+			<Typography children={message} />
+		</div>
+	);
+
+	modalProps.set(ModalProps.propNames.title, titleComponent);
+	modalProps.set(ModalProps.propNames.open, open);
+	modalProps.set(ModalProps.propNames.backdropClickCallback, backdropClickCallback);
+
+	const actionPanel = (
+		<div className={classes.actionPanel}>
+			{actions.map(action => (
+				<Button
+					key={action.label.id}
+					variant={action.isPrimary ? "contained" : "outlined"}
+					color={action.isPrimary ? "primary" : "default"}
+					disableElevation={action.isPrimary}
+					onClick={e => action.handler(e)}
+				>
+					<FormattedMessage {...action.label} />
+				</Button>
+			))}
+		</div>
+	);
+
+	modalProps.set(ModalProps.propNames.actionPanel, actionPanel);
+
+	return <Modal message={messageComponent} modalProps={modalProps} />;
```

Comment:
instead of messageComponent pass the message directly. It will be displayed properly